webpack/assets/javascripts/dashboard/index.js

```diff
@@ -24,7 +24,7 @@ function autoRefresh() {
       const hasFocus = doesDocumentHasFocus();
 
       if (autoRefreshIsOn && hasFocus) {
-        window.Turbolinks.visit(window.location.href);
+        window.location.reload();
```

Comment:
This will be painful I suspect. We default it to off nowadays so I suspect many users won't notice this. Probably deserves some attention.

Reply:
I would guess it was painfull already. Could we just solve it without refreshing the page?

Reply:
I think the dashboard is due for a proper rewrite to React and only refresh relevant content. However, that needs some thought. Most of the graphs there aren't that interesting to refresh regularly because in many use cases they don't change over the course of days. AFAIK we default to not enabling auto refresh now so it doesn't block this effort. Just good to be aware of the bigger picture.